content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -95,6 +95,10 @@ supporting directly installing versions within a given range.
 
 ## Channel Guidelines
 
+#### Channel 
+
+**Definition**: The channel defines a stream of updates for an operator and is used to roll out updates for subscribers. The head points at the latest version of that channel.
```

Comment:
have we not this definition in another place? Why do we need to re-define here?

Reply:
Agreed, it looks like we define a channel in the glossary: https://deploy-preview-131--operator-lifecycle-manager.netlify.app/docs/glossary/#channel. This also feels a bit strange to read given the surrounding context. Seems sufficient to just remove this header and the description entirely.

Reply:
Yeah. Agree. The Channel definition is already part of glossary.

Reply:
Removed it ad this is part of glossary.